objdiff-gui/src/views/function_diff.rs

```diff
@@ -149,6 +149,10 @@ fn ins_hover_ui(
                 ui.colored_label(appearance.highlight_color, "Extern".to_string());
             }
         }
+
+        if let Some(demangled) = rlwinmdec::decode(&ins.formatted) {
+            ui.colored_label(appearance.highlight_color, demangled.trim());
```

Comment:
```suggestion
        if let Some(decoded) = rlwinmdec::decode(&ins.formatted) {
            ui.colored_label(appearance.highlight_color, decoded.trim());
```
nit: `demangled` was copy-pasted and is kind of confusing here